doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -160,6 +160,17 @@ Zigbee
 
 |no_changes_yet_note|
 
+Gaezll
+------
+
+* Added:
+
+   * User-specific file for Gazell pairing. User can override Gazell pair config inside application.
```

Comment:
This changelog line is added in another commit.

Reply:
Separate changelog to 2 commits.